README.md

```diff
@@ -1,2 +1,163 @@
-# 4budget
-Our budget planner and daily budget tracker is a money saving app that makes managing your personal finances simple!
+# 4Budget: Personal Finance Management App
+
+## 📗 Table of Contents
+
```

Comment:
- [ ] Your readme file should contain an Acknowledgement section that gives credit to Gregoire Vella the app designer. Kindly add it.

Reply:
"Thank you for your feedback, @julie-ify ! I just finished it and I'm asking for another review! Greetings!"